src/app/pages/bitcoin-contract-request/components/bitcoin-contract-offer/bitcoin-contract-expiration-date.tsx

```diff
@@ -0,0 +1,24 @@
+import { Flex } from '@stacks/ui';
+
+import { SpaceBetween } from '@app/components/layout/space-between';
+import { Text } from '@app/components/typography';
+
+interface BitcoinContractExpirationDateProps {
+  expirationDate: string;
+}
+export function BitcoinContractExpirationDate({
+  expirationDate,
+}: BitcoinContractExpirationDateProps) {
+  return (
+    <Flex flex={1} width={'284.16px'} paddingLeft={'24px'} paddingRight={'24px'}>
```

Comment:
wonder if there is really need in .16px ?

Reply:
also can use semantic value `loose` instead of 24px